# HG changeset patch
# User nunb <nunb@tallinn>
# Date 1257417321 -3600
# Node ID ba5a806c2b82b272af31b3e2df3b05dec1783c75
# Parent  787ba6f1ef1bbfbc28e688c30429085d7c56104f
Add shadowing import for with-array, which isn't present in the weblocks lisp src.

diff -r 787ba6f1ef1b -r ba5a806c2b82 src/weblocks.lisp
--- a/src/weblocks.lisp	Fri Oct 02 11:29:19 2009 +0200
+++ b/src/weblocks.lisp	Thu Nov 05 11:35:21 2009 +0100
@@ -14,6 +14,7 @@
                             #:standard-class #:typep #:subtypep)
     (:shadowing-import-from :f-underscore #:f #:_)
     (:shadowing-import-from :fare-matcher #:match)
+    (:shadowing-import-from :metabang.utilities #:with-array)
     (:shadow #:redirect #:errors)
     (:documentation
       "Weblocks is a Common Lisp framework that eases the pain of web
# HG changeset patch
# User nunb <nunb@tallinn>
# Date 1257417321 -3600
# Node ID ba5a806c2b82b272af31b3e2df3b05dec1783c75
# Parent  787ba6f1ef1bbfbc28e688c30429085d7c56104f
Add shadowing import for with-array, which isn't present in the weblocks lisp src.

diff -r 787ba6f1ef1b -r ba5a806c2b82 src/weblocks.lisp
--- a/src/weblocks.lisp	Fri Oct 02 11:29:19 2009 +0200
+++ b/src/weblocks.lisp	Thu Nov 05 11:35:21 2009 +0100
@@ -14,6 +14,7 @@
                             #:standard-class #:typep #:subtypep)
     (:shadowing-import-from :f-underscore #:f #:_)
     (:shadowing-import-from :fare-matcher #:match)
+    (:shadowing-import-from :metabang.utilities #:with-array)
     (:shadow #:redirect #:errors)
     (:documentation
       "Weblocks is a Common Lisp framework that eases the pain of web
# HG changeset patch
# User nunb <nunb@tallinn>
# Date 1257417321 -3600
# Node ID ba5a806c2b82b272af31b3e2df3b05dec1783c75
# Parent  787ba6f1ef1bbfbc28e688c30429085d7c56104f
Add shadowing import for with-array, which isn't present in the weblocks lisp src.

diff -r 787ba6f1ef1b -r ba5a806c2b82 src/weblocks.lisp
--- a/src/weblocks.lisp	Fri Oct 02 11:29:19 2009 +0200
+++ b/src/weblocks.lisp	Thu Nov 05 11:35:21 2009 +0100
@@ -14,6 +14,7 @@
                             #:standard-class #:typep #:subtypep)
     (:shadowing-import-from :f-underscore #:f #:_)
     (:shadowing-import-from :fare-matcher #:match)
+    (:shadowing-import-from :metabang.utilities #:with-array)
     (:shadow #:redirect #:errors)
     (:documentation
       "Weblocks is a Common Lisp framework that eases the pain of web
